htdocs/product/card.php

```diff
@@ -918,7 +918,7 @@
         print '</td></tr>';
 
         // Label
-        print '<tr><td class="fieldrequired">'.$langs->trans("Label").'</td><td colspan="3"><input name="label" size="40" maxlength="255" value="'.dol_escape_htmltag(GETPOST('label')).'"></td></tr>';
+        print '<tr><td class="fieldrequired">'.$langs->trans("Label").'</td><td colspan="3"><input name="label" size="60" maxlength="255" value="'.dol_escape_htmltag(GETPOST('label')).'"></td></tr>';
```

Comment:
Using of size="xx" into search field are no more allowed for coming PR.
We must now use class="abcwidthxxx"
when abc is 'min' of 'max' and xxx is 50, 100, 150, 200, depending on need (see css file for complete list of min/maxwidth known.